### PR TITLE
remove unimplemented extensions (was: Update index.bs)

### LINKED ISSUE #1386: Remove unimplemented extensions
At the F2F the work group decided to remove extensions that have not be implemented and are not testable from the Level 2 spec

The proposed list of extensions to be removed are:
Simple Transaction Authorization Extension (txAuthSimple)
Generic Transaction Authorization Extension (txAuthGeneric)
Authenticator Selection Extension (authnSel)
User Verification Index Extension (uvi)
Location Extension (loc)
User Verification Method Extension (uvm)
Biometric Authenticator Performance Bounds Extension (biometricPerfBounds)
and possibly
Supported Extensions Extension (exts)

Feedback on if this is the correct list.
please

### PR DESCRIPTION
**Pull request by ve7jtb**
This removes unimplemented extensions
Fixes #1386

***
<a href="https://pr-preview.s3.amazonaws.com/ve7jtb/webauthn/pull/1399.html" title="Last updated on Apr 9, 2020, 1:37 AM UTC (83092b5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/1399/a636817...ve7jtb:83092b5.html" title="Last updated on Apr 9, 2020, 1:37 AM UTC (83092b5)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -6310,53 +5945,13 @@ IANA "WebAuthn Extension Identifier" registry established by [[!WebAuthn-Registr
...
-    location to the [=[WRP]=], if supported by the [=client platform=] and subject to [=user consent=].
-- Specification Document: Section [[#sctn-location-extension]] of this specification
-    <br/><br/>
 - WebAuthn Extension Identifier: uvm
 - Description: This [=registration extension=] and [=authentication extension=] enables use of a user verification method.
     The user verification method extension returns to the [=[WRP]=] which user verification methods (factors) were
     used for the WebAuthn operation.
 - Specification Document: Section [[#sctn-uvm-extension]] of this specification
     <br/><br/>
-- WebAuthn Extension Identifier: credProps
```
Comment (agl) [inline index.bs:6355]: I don't think this one should be deleted. It's the counterpart to rk=preferred.

Comment (agl) [reply index.bs:6355]: (Based on discussions, this might be correct if this isn't listed by IANA.)

Comment (ve7jtb) [reply index.bs]: I have added credProps back to the IANA registration.

**Comment by equalsJeffH** [comment]
there's another dangling reference: `[[#sctn-location-extension]]`  -- hence the build fails  -- hopefully no more but shud check after fixing this one....

to build spec and get list of error msgs, you can do this on your *nix machine or cygwin terminal:
```
curl https://api.csswg.org/bikeshed/ -F file=@index.bs -F force=1 -F output=err
```

**Comment by equalsJeffH** [comment]
> there's another dangling reference: [[#sctn-location-extension]] -- hence the build fails
I have checked and with the `[[#sctn-location-extension]]`  cross-reference removed the spec builds fine.

I suggest we remove the entire subsection entitled ["13.4.1. Browser Permissions Framework and Extensions"](https://w3c.github.io/webauthn/#sctn-browser-permissions-framework-extensions) because the only extant (in the L1 spec) extension requiring this is the "Location extension", and we are removing it from L2.

**Comment by ve7jtb** [comment]
OK I will remove that section.

**Pull request merged**